community/OM-Operator-Management/cnv-samples/cnv-operator-install/policy-mtv.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: openshift-virtualization-mtv
+  namespace: default
+spec:
+  remediationAction: enforce
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: mtv-namespace
+        spec:
+          remediationAction: enforce
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: "v1"
+                kind: "Namespace"
+                metadata:
+                  name: "openshift-mtv"
+            
```

Comment:
The OperatorPolicy will make this namespace if necessary. Unless you need a special label or something, a separate ConfigurationPolicy isn't needed.

Reply:
Perfect will adjust.